region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -2117,6 +2162,18 @@
         "Do not stand on the Oums when they begin to attack."
       ]
     },
+    {
+      "link": [3, 2],
+      "name": "Use Flash Suit",
+      "requires": [
+        "Morph",
+        "canUseEnemies",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 18, "excessFrames": 11}}
+      ],
+      "flashSuitChecked": true,
+      "note": "Go under the Oums and spark horizontally through the speed blocks, then wait for the Oums and use them as a platform to jump up the last ledge."
```

Comment:
Baiting an oum down to climb up with, and then hitbox clipping the rest is fairly easy to do.  Or you could bring all 3 down. Then its a 4 energy spark.